### ISSUE TITLE
Test suite missing files

### ISSUE THREAD

**Issue by snoyberg** [JustusAdam/marvin#12]
```
> /tmp/stackage-build13/marvin-0.2.1$ dist/build/marvin-integration-tests/marvin-integration-tests

the integration test
  compiles the integration test suite FAILED [1]
initializer
  produces a compileable project FAILED [2]

Failures:

  test/IntegrationSpec.hs:88: 
  1) the integration test compiles the integration test suite
       uncaught exception: IOException of type NoSuchThing (test/integration: getDirectoryContents:openDirStream: does not exist (No such file or directory))

  test/IntegrationSpec.hs:91: 
  2) initializer produces a compileable project
       uncaught exception: ParseException (InvalidYaml (Just (YamlException "Yaml file not found: test/resources/stack.yaml")))

Randomized with seed 1286904463

Finished in 0.0007 seconds
2 examples, 2 failures
```

Likely just need some `extra-source-files`

**Comment by JustusAdam** [comment]
No I just forgot that stack would try to run these tests. They are not meant for stack and hackage, they are sort of internal sanity tests. And I don't want to distribute the files necessary to run them.

I'm working on changing the setup so only travis runs those tests.

**Comment by JustusAdam** [comment]
This should be fixed as of version 0.2.2.

Please confirm and close this issue.

**Comment by JustusAdam** [comment]
I'll close this issue as it seems to be fixed now.
